### PR TITLE
Add missing `bookmark-*` icons

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Review by jguddas — APPROVED** [review]

File: icons/bookmark-off.svg
[the remark this answers is not in the document]
Comment (jguddas) [reply icons/bookmark-off.svg:1]: I agree.

[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/bookmark-off.svg:1]: @karsa-mistmere Removed…

**Comment by ericfennis** [comment]
@danielbayley What is the use case for `bookmark-off`?

**Comment by danielbayley** [comment]
> @danielbayley What is the use case for `bookmark-off`?

@ericfennis Use case was remove bookmark… But not sure we can make the `-off` version work like the others, given the underlying shape… So I replaced it with `-slash`, which could also be _read_. Although actually, I think `-check` would be better for that…

**Review by ericfennis — APPROVED** [review]

**Pull request merged**